test/slurm/run_integration.py

```
@@ -43,7 +43,7 @@
 def run(command: Sequence[str], *, check: bool = True) -> subprocess.CompletedProcess[str]:
     """Run a command and retain output for assertions and diagnostics."""
     LOGGER.info("+ %s", shlex.join(command))
-    result = subprocess.run(command, check=False, capture_output=True, text=True)  # ruff: ignore[subprocess-without-shell-equals-true]
+    result = subprocess.run(command, check=False, capture_output=True, text=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)